.chloggen/fix-remotetap-limit.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I'm a bit concerned about user impact of enabling this. Even though it's clearly documented, users may not be expecting the limit when upgrading here. Should we consider this a breaking change, or assume users will clearly understand what to do here when their perceived performance decreases?

Reply:
> I'm a bit concerned about user impact of enabling this. Even though it's clearly documented, users may not be expecting the limit when upgrading here. Should we consider this a breaking change, or assume users will clearly understand what to do here when their perceived performance decreases?

Thanks for your review. You were right to worry. I've labeled it breaking. I can't think of a better way to inform users about the performance degradation after the upgrade, other than documentation.